docs/wks-and-footloose.md

```diff
@@ -18,19 +18,21 @@
    export BACKEND=docker
    ```
 
-1. Install [`footloose`](https://github.com/weaveworks/footloose):
+1. Install [footloose](https://github.com/weaveworks/footloose):
 
    ```console
    GO111MODULE=on go install github.com/weaveworks/footloose
    ```
 
 1. Start two machines using `footloose`:
 
+These commands assume you are in the `examples/footloose` directory.
+
    ```console
    $ footloose create -c ${DISTRO}/${BACKEND}/singlemaster.yaml
    INFO[0000] Image: quay.io/footloose/centos7 present locally
-   INFO[0000] Creating machine: cluster-node0 ...
-   INFO[0001] Creating machine: cluster-node1 ...
+   INFO[0000] Creating machine: centos-singlemaster-node0 ...
+   INFO[0001] Creating machine: centos-singlemaster-node1 ...
```

Comment:
I'm trying to imagine myself using this on a `ubuntu-multimaster`, and in that scenario it could be confusing that what I'm comparing to in the docs isn't generalized.  If you like this best, let's go with it!  Just something to consider for keeping it generalized.

Reply:
Good point.  This is just output from the command and is specific to the OS which is printed the line before this output.  I'll leave as-is till we have better support for ubuntu.